src/Uno.UWP/ApplicationModel/Appointments/AppointmentStore.Android.cs

```diff
@@ -0,0 +1,179 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Appointments
+{
+	public partial class AppointmentStore
+	{
+		private bool _StartTimeRequested = false; // if it should be included in UWP output result set
+		private bool _DurationRequested = false; // set if duration should be included in UWP output result
+
+		private List<string> UWP2AndroColumnNames(IList<string> uwpColumns)
+		{
+			_StartTimeRequested = false;
+
+			List<string> androColumns = new List<string>();

```

Comment:
```suggestion
			List<string> androidColumns = new();

```

Reply:
Don't resolve the comment if you haven't done the change.

Reply:
But this change was done!